apps/landing/styles.css

```
@@ -0,0 +1,427 @@
+.picker-out-cmd { padding: 18px 16px; font-family: var(--mono); font-size: 12.5px; color: var(--ink); line-height: 1.7; word-break: break-all; }
+.picker-out-cmd .prompt { color: var(--indigo); margin-right: 10px; }
+.picker-out-size { display: flex; align-items: center; justify-content: space-between; padding: 12px 16px; border-top: 1px solid var(--line); background: var(--bg-2); font-family: var(--mono); font-size: 11px; color: var(--ink-3); }
+.picker-out-size strong { color: var(--ink); font-weight: 500; }
+.copy-btn { font-family: var(--mono); font-size: 11px; color: var(--ink-3); background: transparent; border: 1px solid var(--line-2); padding: 4px 10px; border-radius: 3px; }
+.copy-btn:hover { color: var(--ink); border-color: var(--ink); }
+
+/* Schema + browser */
+.support-grid { display: grid; grid-template-columns: 1fr 1fr; gap: 1px; background: var(--line); border: 1px solid var(--line); border-radius: var(--radius-lg); overflow: hidden; }
+@media (max-width: 760px) { .support-grid { grid-template-columns: 1fr; } }
+.support-block { background: var(--bg-card); padding: 28px 24px; }
+.support-block h4 { margin: 0 0 16px; font-size: 14px; font-weight: 500; letter-spacing: 0; font-family: var(--mono); text-transform: uppercase; letter-spacing: 0.06em; color: var(--ink-3); }
```

Stylelint (17.11.1)

[error] Duplicate property "letter-spacing" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)